### ISSUE TITLE
Flat loss

### ISSUE THREAD

**Issue by ajay960singh** [davidtvs/pytorch-lr-finder#10]
Hey guys,
I'm trying to find the optimal range of learning rates for Facial Expression classification but the problem is that I'm getting a flat LR from 1e-6 to 1e-2 and then  it just shoots up and diverges. The flat curve occurs at loss of 1.4, which doesn't seem too low. So does it mean there's a problem with my dataset to it's due to something else?
Thanks

**Comment by NaleRaphael** [comment]
Hi @ajay960singh ,
it's not an easy question to be answered.

I would suggest trying the following approaches to figure it out (without using `LRFinder`):
- Check your model or replace it with a simpler one, then train it directly with a few steps to make sure that loss can decrease.
- Apply your model to other open dataset (e.g. FER2013) and check whether it works normally or not.
- Apply the model architecture provided by others (e.g. Kaggle kernels) on your dataset, and check whether the loss is still not decreasing. If yes, it seems likely there is something wrong in your dataset.

If your model works normally then, it should work fine with `LRFinder`.

Please feel free to let us know if you have any update about this issue. :)

**Comment by davidtvs** [comment]
Closing due to inactivity and lack of enough information to replicate
